modules/ROOT/pages/email/email-gmail.adoc

```diff
@@ -6,23 +6,23 @@ endif::[]
 
 
 [[short_description]]
-The examples here provide guidance on setting up a Gmail connection using Email Connector over the available protocols.
+These examples provide guidance for setting up a Gmail connection using Email Connector over the available protocols.
 
 == Gmail Configurations
 
-By default, Google might block third-party mail clients. For Google to authorize a third-party server to access your account, you must enable `Less Secure Apps` on your Gmail account. You can read a full explanation of the risks involved and get a full explanation https://support.google.com/accounts/answer/6010255[here], which essentially says Google blocks any apps that are not using security protocols that Google deems mandatory unless they allow "less secure apps" to access your Gmail account.
+By default, Google might block third-party mail clients. For Google to authorize a third-party server to access your account, you must enable the *Less Secure Apps* setting for your Gmail account. 
```

Comment:
By default it MIGHT? Does it, or does it not?
Hmm...maybe we could just use the second sentence? Or:
```suggestion
Google might block third-party mail clients if you do not enable the *Less Secure Apps* setting for your Gmail account. 
```